### ISSUE TITLE
Gutenberg: Address Block cannot be read when using a dark theme/background.

### ISSUE THREAD

**Issue by cecoates** [Automattic/jetpack#12738]
#### Steps to reproduce the issue

1. Go to https://wordpress.com/themes/.
2. Activate a theme with a dark background, such as "Sophisticated Business": https://wordpress.com/theme/sophisticated-business.
3. Go to https://wordpress.com/posts/ and edit or create a post.
4. Add the "Address Block".

#### What I expected

To see the fields of the address block, and their contents, legibly.

#### What happened instead
You see several white fields with seemingly no text in them.

You can click on one to add text, but as soon as you click away the text disappears again.

Seems to affect both Jetpack and WP.com sites.

**Screenshots**

![Screen Shot on 2019-06-18 at 14_32_00](https://user-images.githubusercontent.com/10121835/59714881-45b58c00-91d8-11e9-935d-b848fb3ed705.png)

**Comment by jeherve** [comment]
cc @davidakennedy Is this something that should be addressed in Jetpack, or in the theme styles and what styles are applied to input fields in the block editor?

**Comment by jeherve** [comment]
Closing as duplicate of #11853
